css/build.css

```diff
@@ -0,0 +1,419 @@
+body {
+  padding-bottom: 0 !important;
+}
+
+[data-interactive-floorplan-id] {
+  opacity: 1;
+  transition: opacity 0.25s ease-out;
+}
+
+[data-interactive-floorplan-id].is-loading {
+  opacity: 0;
+  pointer-events: none;
+}
+
+.flooplan-app-placeholder {
```

Comment:
You're using this style in a lot of components. Why not use a consistent class instead of one that's specific to Floor Plan, e.g. `.widget-not-configured`